evalml/automl/automl_search.py

```diff
@@ -707,7 +720,7 @@ def _check_for_high_variance(self, pipeline, cv_scores, threshold=0.2):
         if cv_scores_std != 0 and cv_scores_mean != 0:
             high_variance_cv = bool(abs(cv_scores_std / cv_scores_mean) > threshold)
         if high_variance_cv:
-            logger.warning(f"High coefficient of variation (cv >= {threshold}) within cross validation scores. {pipeline_name} may not perform as estimated on unseen data.")
+            logger.warning(f"\t\tHigh coefficient of variation (cv >= {threshold}) within cross validation scores.\n\t\t{pipeline_name} may not perform as estimated on unseen data.")
```

Comment:
Good point. I added the extra tab cause I thought it would be nice to "nest" the pipeline log underneath the batch number during automl search but it's probably equally good to remove the extra tab.